README.md

```diff
@@ -1,6 +1,18 @@
-# Airbrake Golang Notifier [![Build Status](https://travis-ci.org/airbrake/gobrake.svg?branch=v2)](https://travis-ci.org/airbrake/gobrake)
+Gobrake
+=======
 
-<img src="http://f.cl.ly/items/3J3h1L05222X3o1w2l2L/golang.jpg" width=800px>
+[![Build Status](https://travis-ci.org/airbrake/gobrake.svg?branch=v2)](https://travis-ci.org/airbrake/gobrake)
+
+![Gobrake][arthur-go]
+
+Introduction
+------------
+
+_Gobrake_ is a notifier library for [Airbrake][airbrake.io] for the Go
+programming language, the leading exception reporting service. Gobrake provides
+minimalist API that enables the ability to send _any_ Go error or panic to the
+Airbrake dashboard. The library is extremely lightweight and with minimal
+overhead.
```

Comment:
Something about the last sentence doesn't read quite right. The "...and with..." part feels wrong. I think replacing `and` with a comma is better. `The library is extremely lightweight, with minimal overhead.`